tests/testsuite/cargo_report_timings/html_snapshot/out/cargo-timing-20060724T012128000Z-b0fd440798ab3cfb.html

```diff
@@ -195,7 +195,7 @@ <h1>Cargo Build Timings</h1>
 <td>Total time:</td><td>5.6s</td>
 </tr>
 <tr>
-<td>rustc:</td><td>1.94.0-nightly<br>Host: [..]<br>Target: [..]</td>
+<td>rustc:</td><td>rustc 1.94.0-nightly (f52090008 2025-12-10)<br>Host: [..]<br>Target: [..]</td>
```

Comment:
This needs to be redacted

Reply:
Unsure.  I would say it depends on if you are finding it useful

Reply:
Removed. The other alternative is we upload artifact to GitHub, so other can download and review without pulling the PR itself.